hack/generate/samples/generate_testdata.go

```diff
@@ -21,9 +21,9 @@ import (
 
 	log "github.com/sirupsen/logrus"
 
-	"github.com/operator-framework/operator-sdk/hack/generate/samples/internal/ansible"
-	gosamples "github.com/operator-framework/operator-sdk/hack/generate/samples/internal/go"
-	"github.com/operator-framework/operator-sdk/hack/generate/samples/internal/helm"
+	"github.com/operator-framework/operator-sdk/hack/generate/samples/ansible"
+	gosamples "github.com/operator-framework/operator-sdk/hack/generate/samples/go"
+	"github.com/operator-framework/operator-sdk/hack/generate/samples/helm"
```

Comment:
This changes makes the SDK repo have exported APIs, which is against our policy. Refactor this to ensure that the SDK has no exported APIs.

Reply:
We can move the generate to the samples. I would like to keep in the hack because shows fit better. However, it is fine.